.github/workflows/deploy.yml

```diff
@@ -0,0 +1,40 @@
+# yamllint --format github .github/workflows/deploy.yml
+---
+name: deploy
+
+# We deploy on master and release versions, regardless of if the commit is
+# documentation-only or not.
+on:
+  push:
+    tags: '[0-9]+\.[0-9]+\.[0-9]+'
```

Comment:
15.0.1_p9 is an example version. while this is different than normal the above pattern will still match and that's easier to copy/pasta vs making a tighter one here imho

Reply:
This could be a little bit problematic if you decide to publish builds EA jdk builds (`16-ea`) but I think for now it is not an issue.

Reply:
ahh yeah the comments aren't moved into this file, as it is identical for other projects, but shouldn't be a problem as we are using OS packages which are already in a known naming convention ex 15.0.1_p9-r0 https://pkgs.alpinelinux.org/packages?name=openjdk15